packages/editor/src/components/browser/Browser.tsx

```diff
@@ -50,7 +53,11 @@ const Browser = ({ open, onOpenChange, types, accept, location, cmsOptions }: Br
     <>
       <Dialog open={open} onOpenChange={onOpenChange}>
         <DialogTrigger asChild>
-          <Button icon={IvyIcons.ListSearch} aria-label='Browser' />
+          <Button
+            icon={IvyIcons.ListSearch}
+            aria-label='Browser'
+            onClick={() => initSearchFilter !== undefined && setInitSearchFilterValue(initSearchFilter)}
```

Comment:
I was initially also a bit skeptical about whether this would work, but it with fortunatly the onClick worked well.

I tried it this way, but unfortunately, it didn't work. I believe the issue is that I lose the selection after clicking the button to open the browser. Therefore, when I pass the initSearchFilter function to useTypeBrowser and call it during the initialization of the browser, it returns an empty string because the selection has already been cleared by the button click.

Reply:
Nevermind,  it worked as you mentioned. I made a mistake during testing and tried it on a ScriptInput instead of a ScriptArea :-)